addons/point_of_sale/views/pos_payment_method_views.xml

```diff
@@ -18,7 +18,7 @@
                         <group>
                             <field name="split_transactions"/>
                             <field name="journal_id" required="not split_transactions" placeholder="Leave empty to use the receivable account of customer" />
-                            <field name="outstanding_account_id" groups="account.group_account_readonly" invisible="type != 'bank'" placeholder="Leave empty to use the default account from the company setting" />
+                            <field name="outstanding_account_id" groups="account.group_account_readonly" invisible="type != 'bank'" required="type == 'bank'" placeholder="Selection of outstanding account is required" />
```

Comment:
https://github.com/odoo/odoo/wiki/Contributing#how-should-i-handle-translatable-content

Reply:
>`New releases will have a stabilization period of a couple of month when modifications of source terms is tolerated.`

So should we not change placeholder and labels in `18.0` ?

Reply:
Ideally we should change that in master